include/boost/math/special_functions/detail/bessel_ik.hpp

```diff
@@ -77,10 +77,10 @@ inline T bessel_i_small_z_series(T v, T x, const Policy& pol)
    return prefix * result;
 }
 
-// Calculate K(v, x) and K(v+1, x) by method analogous to
+// Calculate result_K(v, x) and result_K(v+1, x) by method analogous to
```

Comment:
Nit pick: K here refers to the Bessel function K, not to our variable, so I think we should leave the comment unchanged.